### PR TITLE
Check Association Inverses

### LINKED ISSUE #199: Install schema_plus_association_inverses

### PR DESCRIPTION
**Pull request by lowjoel**
Also involves a `bundle update`.

Fixes #199.

### CODE REVIEW

File: app/views/user/registrations/edit.html.slim [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  = t('.edit', resource: resource_name.to_s.humanize)
+
+= simple_form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f|
+  = f.error_notification
+
+  div.form-inputs
+    = f.input :email, required: true, autofocus: true
+
+    - if devise_mapping.confirmable? && resource.pending_reconfirmation?
+      p Currently waiting confirmation for
```
Comment (allenwq) [inline app/views/user/registrations/edit.html.slim]: Missed translations here

**Comment by lowjoel** [comment]
@allenwq done, thanks.

File: app/views/user/registrations/edit.html.slim [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+    = f.input :password, autocomplete: 'off', hint: t('.new_password_hint'), required: false
+    = f.input :password_confirmation, required: false
+    = f.input :current_password, hint: t('.current_password_required_hint'), required: true
+
+  div.form-actions
+    = f.button :submit
+
+h3
+  | Cancel my account
```
Comment (allenwq) [inline app/views/user/registrations/edit.html.slim]: sorry, all the texts in this file are not translated.

File: app/views/user/registrations/edit.html.slim [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+p
+  | Unhappy?
```
Comment (allenwq) [inline app/views/user/registrations/edit.html.slim]: Here

File: app/views/user/registrations/edit.html.slim [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  = link_to 'Cancel my account', registration_path(resource_name), data: { confirm: 'Are you sure?' }, method: :delete
+
+= link_to 'Back', :back
```
Comment (allenwq) [inline app/views/user/registrations/edit.html.slim]: And the link here

**Comment by lowjoel** [comment]
OK. That file actually was generated by Devise.

File: app/views/user/registrations/edit.html.slim [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    = f.input :current_password, hint: t('.current_password_required_hint'), required: true
+
+  div.form-actions
+    = f.button :submit
+
+h3
+  | t('.cancel_account')
+
+p
+  | Unhappy?
```
Comment (allenwq) [inline app/views/user/registrations/edit.html.slim]: @lowjoel here still got one

**Comment by lowjoel** [comment]
done.

**Pull request merged**